src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationTracker.State.cs

```diff
@@ -37,8 +37,10 @@ private class State
                 public Compilation? DeclarationOnlyCompilation { get; }
 
                 /// <summary>
-                /// The best compilation that is available that source generators have not ran on. May be an in-progress, full declaration,
-                /// a final compilation, or <see langword="null"/>.
+                /// The best compilation that is available that source generators have not ran on. May be an in-progress,
```

Comment:
might be worth renaming this property then to make this more apparent.

Reply:
Good idea, renamed.